pytorch_lightning/callbacks/quantization.py

```diff
@@ -204,6 +244,11 @@ def on_fit_start(self, trainer, pl_module):
         # the model that will observe weight and activation tensors during calibration.
         torch.quantization.prepare_qat(pl_module, inplace=True)
 
+        fake_quants = tuple(
+            module for module in pl_module.modules() if isinstance(module, torch.quantization.FakeQuantizeBase)
```

Comment:
noob question. Why does PyTorch introduce FakeQuantize Layer ?

Reply:
I'm not sure but maybe PyTorch `FakeQuantizeBase` defines the interface of classes deriving the quantization range and applying the fake quantization. And currently there are three children of `FakeQuantizeBase`:

- `FixedQParamsFakeQuantize`: the quantization range is provided by users. 
- `FakeQuantize`: the quantization range is statistically derived from the data.
- `_LearnableFakeQuantize`: the quantization range is learned through backpropagation.

Maybe `FakeQuantize` could be considered as `CalibrationFakeQuantize`?